src/components/Trade/TradeInput.tsx

```diff
@@ -284,7 +284,7 @@ export const TradeInput = () => {
         return
       }
       const trade = await getTrade()
-      if (trade.isErr()) throw new Error(trade.unwrapErr().message)
+      if (trade.isErr()) throw trade.unwrapErr()
```

Comment:
is this actually a native js `Error`, or an object?

Reply:
Nah it's a `SwapErrorRight` object, in an error-like shape